README.md

```diff
@@ -1,2 +1,70 @@
 # Telnyx Video SDK for iOS
-This repository contains releases of Telnyx's Video SDK for iOS. These releases can be installed as a CocoaPod or manually into your project.
+This repository contains releases of Telnyx's Video SDK for iOS. Currently a release is installed through CocoaPods.
+
+## Adding Telnyx Video SDK to your iOS Client Application:
+Currently the Video SDK is delivered through Cocoapods. 
+
+### Cocoapods
+
+If your xcode project is not using [cocoapods](https://cocoapods.org/) yet, you will need to configure it.
+
+1. Open your podfile and add the TelnyxVideoSdk
+
+```
+pod 'TelnyxVideoSdk', '~> 0.0.3'
+```
+
+2. At the end of your podfile also add the following post_install sequence:
+
+```
+post_install do |installer|
+    installer.pods_project.targets.each do |target|
+      target.build_configurations.each do |config|
+        config.build_settings['BUILD_LIBRARY_FOR_DISTRIBUTION'] = 'YES'
+      end
+    end
+end
+```
+
+Your podfile should look something like this: 
+
+```
+platform :ios, '12.0'
+
+target 'YOUR_APP_TARGET' do
+  use_frameworks!
+
+  # Pods for YOUR_APP_TARGET
+  pod 'TelnyxVideoSdk', '~> 0.0.1'
+
+  post_install do |installer|
+    installer.pods_project.targets.each do |target|
+      target.build_configurations.each do |config|
+        config.build_settings['BUILD_LIBRARY_FOR_DISTRIBUTION'] = 'YES'
+      end
+    end
+  end
+end
+```
+
+3. Install your pods. You can add the flag --repo-update to ensure your cocoapods has the specs updated.
+
+```Bash
+pod install --repo-update
```

Comment:
@guillermobattistel 
`pod install --repo-update` does not install the latest version of the dependency. We need to use `pod update` or  `pod update TelnyxVideoSdk`